editor/debugger/editor_visual_profiler.cpp

```diff
@@ -70,7 +70,7 @@ void EditorVisualProfiler::add_frame_metric(const Metric &p_metric) {
 	updating_frame = true;
 	clear_button->set_disabled(false);
 	cursor_metric_edit->set_max(frame_metrics[last_metric].frame_number);
-	cursor_metric_edit->set_min(MAX(frame_metrics[last_metric].frame_number - frame_metrics.size(), 0u));
+	cursor_metric_edit->set_min(MAX(static_cast<int64_t>(frame_metrics[last_metric].frame_number) - frame_metrics.size(), 0u));
```

Comment:
Wouldn't this work too?

```suggestion
	cursor_metric_edit->set_min(MAX(int64_t(frame_metrics[last_metric].frame_number) - frame_metrics.size(), 0u));
```

Reply:
That works too but I thought `static_cast<>` was the preferred way to do it. As I said, I'm bad at C++.

Reply:
I have updated the PR to use the style of casting that you proposed.

I searched through the codebase and `int64_t()` is much more commonly used than `static_cast<int64_t>`. I also checked some C++ reference docs and semantically both should be identical when it comes to numeric data types.